examples/node/Examples/mongodb.ts

```diff
@@ -38,10 +38,10 @@ async function getPlantsCollection() {
   > = Realm.Services.MongoDB.MongoDBCollection<T>;
   // :code-block-end:
   // :code-block-start: plants-collection-handle
-  const mongodb: Realm.Services.MongoDB = app.currentUser.mongoClient(
+  const mongodb = app.currentUser.mongoClient(
     "mongodb-atlas"
   );
-  const plants: MongoDBCollection<Plant> = mongodb
+  const plants = mongodb
     .db("example")
     .collection("plants");
```

Comment:
This should specify the generic argument to provide the shape of the returned objects.

Reply:
```suggestion
    .collection<Plant>("plants");
```
^ Per Nikola's suggestion